test/provision_test.py

```diff
@@ -168,18 +154,46 @@ def test_provision_with_sealed_container(self):
         assert expected_instruction_as_json == actual_instruction_as_json
 
     def test_for_multiple_dispenses_of_resource_in_containers_larger_than_900ml(self):
-        p = Protocol()
+        self.p.refs.clear()
         w1 = (
-            p.ref("w1", None, cont_type="micro-2.0", discard=True)
+            self.p.ref("w1", None, cont_type="micro-2.0", discard=True)
             .well(0)
             .set_volume("2:microliter")
         )
-        p.provision("rs17gmh5wafm5p", w1, "1500:microliter")
+        self.p.provision("rs17gmh5wafm5p", w1, volumes="1500:microliter")
 
         actual_instruction_as_json = json.dumps(
-            p.as_dict()["instructions"], indent=2, sort_keys=True
+            self.p.as_dict()["instructions"], indent=2, sort_keys=True
         )
         expected_instruction_as_json = TestUtils.read_json_file(
             "split_provisions_by_volume.json"
         )
         assert expected_instruction_as_json == actual_instruction_as_json
+
+    def test_provision_well_with_mass(self):
+        self.p.provision("rs17gmh5wafm5p", self.w1, amounts="50:ug")
```

Comment:
nit: we're missing a test case that correctly provisions more than one mass

Reply:
Added explicit test case `test_provision_multiple_wells_with_diff_masses`